app/Repositories/User/UserRepository.php

```diff
@@ -54,6 +54,17 @@ public function searchByName($name)
         return $this->model->name($name)->get();
     }
 
+    /**
+     * Search by email
```

Comment:
Perhaps, update the function description to "Search by name and email", what do you think?

Reply:
Updated.